src/RA_Defs.h

```diff
@@ -34,7 +36,6 @@
 #define NOSYSCOMMANDS
 #define OEMRESOURCE
 #define NOATOM
-#define NOCLIPBOARD
```

Comment:
When this was put in the pch it gave an error. We probably won't need it later when using HeapAlloc instead of GlobalAlloc.